lib/PuppeteerSharp/ConsoleType.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PuppeteerSharp
+{
+    public enum ConsoleType
```

Comment:
I this should have more options - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/puppeteer/index.d.ts#L133